### ISSUE TITLE
Send logs from the container in case that it fails to update

### ISSUE THREAD

**Issue by botzill** [keel-hq/keel#380]
I was wondering if such a feature could be useful. When we update the image and there are errors(from various reason) send standard k8s logs but as well take last 100 lines from the container log and send them as well to slack. I know that this may be the scope of smth like fluent-bit or other tools especially for this, but I find this useful to have.

Thx.

**Comment by rusenask** [comment]
Indeed, I would really want to expand bot capabilities as well. Getting logs would be nice :)

**Comment by botzill** [comment]
Great! looking forward.
